content/en/docs/userguide/try_fault_injection.md

```diff
@@ -0,0 +1,121 @@
+---
+draft: false
+linktitle: Try Fault Injection
+menu:
+  docs:
+    parent: user guide
+    weight: 10
+title: Try Fault Injection
+toc: true
+type: docs
+
+
+---
+
+### Preparation
+
+1. Make default namespace managed by Kmesh
+
+2. Deploy bookinfo as sample application and sleep as curl client
+
+3. Install service granularity waypoint for reviews and ratings service
```

Comment:
```suggestion
3. Install service granularity waypoint for reviews service
```
We'd better list the installation stesp of `ratings` waypoint directly, because it's easy to overlook and it may not be that easy for first-time users.

Reply:
@noobwei only display installation steps of `ratings` waypoint here as step 4, the installation of `reviews` waypoint has been well explained in `Install Waypoint`.